src/drivers/vmount/output_mavlink.cpp

```diff
@@ -54,7 +54,7 @@ OutputMavlink::OutputMavlink(const OutputConfig &output_config)
 
 int OutputMavlink::update(const ControlData *control_data)
 {
-	vehicle_command_s vehicle_command;
+	vehicle_command_s vehicle_command = {};
```

Comment:
I can do that but keep in mind that this type of initialization + code generator that do struct alignment could cause several errors like this:
```px4-firmware/src/modules/commander/commander.cpp:4098:2:` sorry, unimplemented: non-trivial designated initializers not `supported```
if one variable is added to the struct and it changes the order.